### DISCUSSION TITLE
LMS on webspace

### DISCUSSION THREAD

**Discussion by Outlaw187** [LMS-Community/slimserver#1023] [Q&A]
Hello, I am new to this forum and quite unexperienced when it comes to github files. I came along the LMS slim server files and already uploaded them to my workspace. But how do I make it run? I want to continue using my squeezebox radio despite they shut down the servers. Any support is highly appreciated. Thanks

**Comment by michaelherger** [comment]
I'd recommend you head over to https://forums.slimdevices.com/forum/user-forums/general-discussion. There's much more information over there, and many more helpful people.

**Reply by Outlaw187** [reply]
Thanks for your honest opinion
